src/Page/Food/Builder.elm

```diff
@@ -131,6 +135,7 @@ init ({ db, builderDb, queries } as session) trigram maybeQuery =
               , comparisonUnit = ComparatorView.PerKgOfProduct
               , displayChoice = ComparatorView.IndividualImpacts
               , modal = NoModal
+              , hovering = []
```

Comment:
Same, maybe `chartHovering` or `comparativeChartHovering`?

Reply:
Renamed to `chartHovering`